plugin.video.latelelibre_fr/resources/lib/lutil.py

```diff
@@ -1,74 +1,80 @@
 # _*_ coding: utf-8 _*_
 
 '''
-   lutil: library functions for XBMC video plugins.
-   Copyright (C) 2013 José Antonio Montes (jamontes)
+   lutil: library functions for KODI media add-ons.
+   Copyright (C) 2017 José Antonio Montes (jamontes)
 
    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.
-   
+
    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.
-   
+
    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
    Description:
    These funtions are called from the main plugin module, aimed to ease
    and simplify the plugin development process.
-   Release 0.1.7
+   Release 0.1.10
 '''
 
 # First of all We must import all the libraries used for plugin development.
 import re, urllib, urllib2
+from datetime import date
 
 debug_enable = False # The debug logs are disabled by default.
 
-def set_debug_mode(debug_flag):
-    """This function sets the debug_enable var to log everything if debug option is true."""
-    global debug_enable
-    debug_enable = debug_flag in ("true", True)
 
+def local_log(message):
+    """This function logs the messages into the main KODI log file. Called from the libraries module by other functions."""
 
-def log(message):
-    """This function logs the messages into the main XBMC log file. Called from main plugin module."""
     if debug_enable:
         print "%s" % message
```

Comment:
please replace this print statement with `xbmc.log` method with the log level set to `xbmc.LOGDEBUG`(see: http://mirrors.kodi.tv/docs/python-docs/13.0-gotham/xbmc.html).
Example: `xbmc.log(msg='This is a test string.', level=xbmc.LOGDEBUG));`.

You can also drop the if condition and the addon debug setting all together. It looks a bit more sane to enable debugging on a single location (on Kodi itself) than having debug settings hidden in the code or in the addon settings.